tests/src/test/scala/org/virtuslab/tests/BazelOpenProjectFixture.scala

```diff
@@ -0,0 +1,30 @@
+package org.virtuslab.tests
+
+import com.intellij.remoterobot.fixtures.CommonContainerFixture
+import org.virtuslab.ideprobe.RunningIntelliJFixture
+import org.virtuslab.ideprobe.protocol.ProjectRef
+import org.virtuslab.ideprobe.robot.RobotPluginExtension
+
+trait BazelOpenProjectFixture extends BazeliskExtension { this: RobotPluginExtension =>
+  implicit class SearchableComponentExt(sc: CommonContainerFixture) {
+    def doClick(): Unit = sc.runJs("component.doClick();", true)
+    def setText(text: String): Unit = sc.runJs(s"component.setText('$text');", true)
+  }
+
+  def openProjectWithBazel(intelliJ: RunningIntelliJFixture): ProjectRef = {
+    val path = intelliJ.workspace.toRealPath()
+    intelliJ.probe.setupBazelExec(bazelPath(intelliJ.workspace).toString)
```

Comment:
does user have to do this?

Reply:
only if there's no `bazel` bin in PATH

Reply:
This is for later, but I remember adding a way to pass custom env to intellij when it is started, through fixture configuration. We might use it to put bazel on PATH, and simulate the actual scenario more closely.